### DISCUSSION TITLE
How would I add `set relativenumber` to the main config?

### DISCUSSION THREAD

**Discussion by pianoboy71** [NvChad/NvChad#2981] [Q&A] [answered]
I see that `set number` has been enabled somewhere, but I can't find where it is for me to add `set relativenumber` as well

**Comment by srodriguez91** [comment]
`:set rnu` should trigger it for you

**Reply by pianoboy71** [reply]
I meant for it to mean that i didn't know which file to paste the `:set relativenumber` (or `:set rnu`) command in. In my main Neovim config, there was an 'init.vim' file to add it to, but in NvChad there's "init.lua", "options.lua" and "lspconfig.lua", and i think there are more .lua files, and i'm not sure which one I should add relativenumber to.

**Reply by srodriguez91** [reply, marked as answer]
Gotcha you can set it from any buffer in nvim and it should persist but you can also set it in your init.lua by adding the line `vim.o.relativenumber =true` you can refer to `:help vim.wo` for more info.

**Reply by Efojensen** [reply]
Thanks. It worked for me.

**Reply by siduck** [reply]
> Thanks. It worked for me.

put `vim.o.rnu=true` in your options.lua file

**Comment by dangdd2003** [comment]
There is a file in the nvchad starter called `lua/options.lua` - where all neovim options are set, add this line to it
```lua
vim.opt.relativenumber = true
```
or
```lua
vim.o.relativenumber = true
```
(if u ask about the difference, as i know, `o` requires option with 'string', `opt` requires option in a table, for boolean, no difference)
The option to enable line number has already been set in `NvChad/lua/nvchad/options.lua` in NvChad repo
